warehouse/internal/repo/schema.go

```diff
@@ -42,15 +45,14 @@ func NewWHSchemas(db *sqlmiddleware.DB, opts ...Opt) *WHSchema {
 	return r
 }
 
-func (sh *WHSchema) Insert(ctx context.Context, whSchema *model.WHSchema) (int64, error) {
-	var (
-		id  int64
-		now = sh.now()
-	)
+// Insert inserts a schema row in wh_schemas with the given schema.
+// If enableTableLevelSchema is true, it also inserts/updates table-level schemas for each table in the schema.
+func (sh *WHSchema) Insert(ctx context.Context, whSchema *model.WHSchema, enableTableLevelSchema bool) error {
```

Comment:
Instead of passing `enableTableLevelSchema` as an argument, should be add it in the `WHSchema` struct via constructor ?